src/NHibernate.Test.VisualBasic/Program.vb

```diff
@@ -4,4 +4,4 @@ Public Class Program
         Return New NUnitLite.AutoRun(GetType(Program).Assembly).Execute(args)
     End Function
 End Class
-#End If
\ No newline at end of file
+#End If
```

Comment:
There was changes required for enabling test under VS with 5.3.x branch, which are no needed on master. Remain some white spaces changes due to editorconfig normalization. That is the same thing for other files with no significant changes like project files.